metron-interface/metron-alerts/e2e/alerts-list/tree-view/tree-view.po.ts

```diff
@@ -97,21 +155,24 @@ export class TreeViewPage {
   expandSubGroupByPosition(groupName: string, rowName: string, position: number) {
     let subGroupElement = element.all(by.css('[data-name="' + groupName + '"] tr[data-name="' + rowName + '"]')).get(position);
     return waitForElementVisibility(subGroupElement)
-    .then(() => browser.actions().mouseMove(subGroupElement).perform())
-    .then(() => subGroupElement.click());
+    .then(() => scrollIntoView(subGroupElement.element(by.css('.fa-caret-right')), true))
+    .then(() => subGroupElement.click())
+    .then(() => waitForElementVisibility(subGroupElement.element(by.css('.fa-caret-down'))));
   }
 
-  getDashGroupTableValuesForRow(name: string, rowId: number) {
-    this.scrollToDashRow(name);
-    return waitForElementPresence(element(by.css('[data-name="' + name + '"] table tbody tr'))).then(() => {
-      return element.all(by.css('[data-name="' + name + '"] table tbody tr')).get(rowId).all(by.css('td')).getText();
-    });
-  }
+  // getDashGroupTableValuesForRow(name: string, rowId: number) {
```

Comment:
There are still blocks of code commented out that need cleaned up.  Maybe you didn't push some additional commits out?

Reply:
@nickwallen No, this looks like something I need to address. Thanks for pointing this out. Previously, I only addressed the specific lines that were called out before in comments. I'll run through all currently commented tests in this PR, update and have @iraghumitra review since this stems from the work he did. I also see some console.log's in there that I'll remove.